site/ai-playground/src/app.tsx

```diff
@@ -75,7 +75,7 @@ const App = () => {
 
   const agent = useAgent<Playground, PlaygroundState>({
     agent: "playground",
-    name: sessionId,
+    name: "Cloudflare-AI-Playground",
```

Comment:
This would make every client connect to the same agent instance. Maybe it can be Cloudflare-Ai-playground-:id?

Reply:
Done!